src/ui/MainRootWindow.qml

```diff
@@ -497,7 +497,6 @@ ApplicationWindow {
                 width:              (backTextLabel.x + backTextLabel.width) - backIcon.x
                 onClicked: {
                     toolDrawer.visible      = false
-                    toolDrawer.toolSource   = ""
```

Comment:
Why is this removed? This means the drawer won't be freed from memory.

Reply:
You are totally right, sorry. I redid this using a timer.